server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -148,26 +148,30 @@ public static Club getClub(DatabaseClient dbClient, String clubId) {
   }
 
   /**
-  * Runs a transaction that adds a membership to the database.
+  * Runs a transaction that adds a membership or ownership to the database.
   * This method checks if a person is already a member of a club by calling a helper function.
   * If the person does not exist, this method will buffer a single mutation that adds
-  * the membership. Otherwise, it will throw an exception indicating that the person
+  * the membership (or owner). Otherwise, it will throw an exception indicating that the person
   * is already a member of the club.
   *
-  * @param  dbClient    the database client
-  * @param  userId      the user ID string used to perform the transaction
-  * @param  clubId      the club ID string used to perform the transaction
+  * @param  dbClient             the database client
+  * @param  userId               the user ID string used to perform the transaction
+  * @param  clubId               the club ID string used to perform the transaction
+  * @param  membershipLevel      the integer representing membership level (member or owner)
   */
-  public static void runAddMembershipTransaction(DatabaseClient dbClient, String userId, String clubId) {
+  public static void runAddAnyMembershipTypeTransaction(DatabaseClient dbClient, String userId,
+                                                            String clubId, int membershipLevel) {
     dbClient
         .readWriteTransaction()
         .run(
           new TransactionCallable<Void>() {
             @Override
             public Void run(TransactionContext transaction) throws Exception {
-              Boolean exists = StorageHandlerHelper.checkMembership(transaction, userId, clubId);
+              Boolean exists = StorageHandlerHelper.checkAnyMembership(transaction, userId, clubId);
```

Comment:
`exists` will be true for the case where the person is a member of the club, but not an owner. Do we want to return `PERSON_ALREADY_IN_CLUB` in that case, or should we update the membershipType.

Reply:
I believe the way I implemented `checkAnyMembership`, it should return true in the case that any membership (whether it be of type owner or member) exists in the Memberships table, and thus will set `exists` to true. The tests I have written for this function suggest that it works for both member and owner.

Reply:
My concern is the case where the person is currently a member, but is trying to become an owner. Can we add a test case for that?

Reply:
Currently in the DB, the memberships table has a key of (userId, clubId), so it will be impossible to add a membership of type owner, since the (userId, clubId) combination already exists. I added a test to verify that an exception is thrown as expected when trying to add a current member as an owner to the DB.